up-l2/rpcclient.adoc

```diff
@@ -39,10 +39,10 @@ SPDX-License-Identifier: Apache-2.0
 
 | `IN` | Input parameter
 | `OUT`| Output parameter
```

Comment:
OUT was meant to be a passed parameter that is an output and not the return value from a method but you prefer to refer to IN as a parameter and out as the return value from the function?

Reply:
To be honest: I have no clue. The semantics of IN, OUT, IN/OUT are, at best, unclear to me.

I had already thought about proposing to use an IDL that has some defined semantics instead of _pseudo code_. Maybe using UML2 semantics or just using Java Interfaces and Methods?